docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -127,7 +127,7 @@ Create the role that will allow a user to connect to any database using any
 database account:
 
 ```sh
-$ tctl --config=/path/to/teleport.yaml create <<EOF
+$ tctl --config=/path/to/teleport-db-role.yaml create <<EOF
```

Comment:
I changed this name, so people don't confuse it with `teleport.yaml` config file.